include/classes/AMZNBS/Admin.class.php

```diff
@@ -161,7 +162,8 @@ private function init_install_options() {
         'install_date'    => null,
         'upgrade_date'    => null),
       'default' => null,
-      'dynamic' => array()
+      'dynamic' => array(),
+      'default_meta' => array()
```

Comment:
This new key stores the meta info about the asins, including the title and image.